cafebabel/templates/users/detail.html

```diff
@@ -10,7 +10,7 @@ <h2>About</h2>
   {% if user.profile.has_image %}
   <img src={{ user.profile.image_url }} alt="Picture of {{ user }}">
   {% else %}
-  <img src=/static/img/profile.svg width=200px height=200px alt="Blank profile picture">
+  <img src=/static/img/profile.svg alt="Default profile picture">
```

Comment:
We should have a template tag for that in order to avoid the repetition here and then.